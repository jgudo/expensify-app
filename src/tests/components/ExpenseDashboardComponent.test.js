import React from 'react';
import {shallow} from 'enzyme';
import ExpenseDashboardComponent from '../../components/ExpenseDasboardComponent';

test('Should render dashboard page correctly', () => {
    const wrapper = shallow(<ExpenseDashboardComponent />);
    expect(wrapper).toMatchSnapshot();
});