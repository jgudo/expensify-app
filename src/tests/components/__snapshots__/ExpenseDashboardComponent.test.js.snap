// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render dashboard page correctly 1`] = `
<div>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
